src/ansys/mechanical/core/embedding/app_libraries.py

```diff
@@ -51,11 +51,9 @@
 import os
 import sys
 
-from ansys.tools.path.path import _get_unified_install_base_for_version
 
-
-def add_mechanical_python_libraries(version: int):
```

Comment:
for backward compatibility, make the argument app_or_version and take both types. If a version is supplied, add an obsolete warning